Add homebrew step to CLI release (#444)
kyma-project/community: guidelines/releases-guidelines/03-kyma-cli-release-process.md

5. Push the tag to trigger a postsubmit job that creates the GitHub release. Validate if the release is available under [releases](https://github.com/kyma-project/cli/releases).

6. Once the release is complete, remove the existing `stable` tag and re-create it pointing to the latest commit on the `master` branch. Pushing this tag will trigger a postsubmit job that builds and pushes the latest stable binaries to a storage bucket. These are used by the continuous integration jobs.

> 7. Update the Homebrew formula of the CLI to the just released version by following this [guide](https://github.com/Homebrew/brew/blob/master/docs/How-To-Open-a-Homebrew-Pull-Request.md).
>    See an example homebrew Kyma CLI formula version bump here: https://github.com/Homebrew/homebrew-core/pull/52375

Reviewer: ```suggestion
7. Update the Homebrew formula of the CLI to the just-released version by following this [guide](https://github.com/Homebrew/brew/blob/master/docs/How-To-Open-a-Homebrew-Pull-Request.md). For a sample Homebrew Kyma CLI formula version bump see the information in [this](https://github.com/Homebrew/homebrew-core/pull/52375) PR.
```
